### DISCUSSION TITLE
Feature request: (bulk) in-addr creation (after the fact)

### DISCUSSION THREAD

**Discussion by hvisage** [poweradmin/poweradmin#426] [Ideas]
I have a few cases where IP addresses had to change. When adding an A record (bulk adding would be the next feature request ;) ) there is a check box to create the reverse.

Might be great to have a checkbox in the zone list, from where you could do things like bulk delete, reverse creation etc.

**Comment by edmondas** [comment]
@hvisage Did you mean to create or delete reverse zones? Creating reverse would require additional data. The reverse zone may also be different.

**Reply by hvisage** [reply]
create the reverse *record* (in the best matching zone?) I believe the code is there, just to also "add" it when modifying/updating an A record

**Comment by edmondas** [comment]
I believe this feature is already implemented, so I'm closing this issue. If you're still experiencing problems or have additional concerns, please feel free to create a new issue with more specific details. Thanks for your contribution!

<img width="1319" alt="Screenshot 2025-04-18 at 09 54 08" src="https://github.com/user-attachments/assets/3ebb7b75-d635-48f6-a913-e6601e88f030" />

**Comment by edmondas** [comment]
Bulk deleting of zones from the zone list will be also in the next version (4.0.0), and PTR record generator added to the same list.

<img width="1297" alt="Screenshot 2025-04-19 at 17 26 42" src="https://github.com/user-attachments/assets/bf15c7b9-cf9e-44f7-aafa-50a93b94370c" />
